src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
-'use strict';
+const succsesFunc = (message) => {
```

Comment:
The function name `succsesFunc` seems to be a typo. Consider renaming it to `successFunc` for better readability and to avoid confusion.